### PR TITLE
[merged] commctl: Add 'list clusters' and 'list hosts' commands

### PR DESCRIPTION
**Pull request by mbarnes**
In playing with `commctl` last week, the very first thing I tried was querying what clusters and hosts I had already set up from previous test sessions.  Seems the tool is missing this, so I'm plugging the gap with some new commands.

`commctl list clusters` takes no arguments.

`commctl list hosts` takes an optional `-n` / `--name` argument, which restricts the list to the given cluster name.

I opted for a new verb `list` instead of extending `get` because I felt it was more intuitive.  Whereas `get` will return detailed status info, `list` will return a compact list with no details.

### CODE REVIEW

**Comment by ashcrow** [comment]
Reviewing now.

**Comment by ashcrow** [comment]
:+1: to adding ```list```.

File: src/commissaire/client_script.py [hunk 1]
```diff
@@ -167,6 +167,33 @@ def create_upgrade(self, name, **kwargs):
...
+        :param kwargs: Keyword arguments
+        :type kwargs: dict
+        """
+        uri = '{0}/api/v0/clusters'.format(self.endpoint)
+        return self._get(uri)
+
+    def list_hosts(self, name, **kwargs):
+        """
+        Attempts to list all hosts or hosts in particular cluster.
+
```
Comment (ashcrow) [inline src/commissaire/client_script.py]: nit: ```name``` isn't documented.

**Comment by ashcrow** [comment]
:+1: Works as expected.

@mbarnes do you want to update for the nit or push it to when we decide on the official location for the CLI?

**Comment by ashcrow** [comment]
@cgwalters-bot try

**Comment by mbarnes** [comment]
:arrow_up:  Nit update.

**Comment by mbarnes** [comment]
On a related note, I find I'm consistently forgetting to type `-n` in `commctl` commands because it feels like the cluster name should be a positional argument, especially since it's almost always required (options are for optional stuff).  What do you think about changing the syntax?

**Comment by ashcrow** [comment]
@mbarnes I'm down with changing the syntax, however, the syntax will probably go through a change via [this spike](https://github.com/projectatomic/commissaire/wiki/SPIKE:-Find-the-best-location-for-the-CLI). If the outcome is to keep going the same direction we are going I'm in 100% agreement it should change to positional.

**Comment by ashcrow** [comment]
@cgwalters-bot try

**Comment by ashcrow** [comment]
@cgwalters-bot r+ a152df6465c3b616135d0d36fe12a8a3fe811987

**Pull request closed without merging**